src/reactive/value.rs

```diff
@@ -3604,16 +3614,16 @@ pub enum Validation {
     /// The data is valid.
     Valid,
     /// The data is invalid. The string contains a human-readable message.
-    Invalid(String),
+    Invalid(Value<MaybeLocalized>),
```

Comment:
Why is this a `Value` rather than just a `MaybeLocalized`? Have you made sure that everywhere this is used that the contained value is properly being tracked to ensure external updates invalidate where it is being used?